CONTRIBUTING.md

```diff
@@ -104,41 +102,55 @@ variables:
 
 `.\script\configure-integration-tests.ps1`
 
-Once these are set, the integration tests will be executed both when 
-running the IntegrationTests build target, or when running the 
-Octokit.Tests.Integration assembly in the Visual Studio test runner.
+After running this, ensure any existing instances of Visual Studio are restarted
+so they pick up the new environment variables are detected.
+
+With these variables set, you can run the integration tests locally using
+`.\build IntegrationTests` or by running the `Octokit.Tests.Integration`
+assembly in the Visual Studio test runner.
+
+**Note:** as the integration tests rely on using the actual GitHub API, you may
+encounter issues if running the tests too frequently. Please use a test account
+so that the unlikely scenario of your account being identified as a spammer.
+
+### Testing Documentation
+
+If you are making changes to the documentation for Octokit, you can test these
+changes locally using the [guide](https://github.com/shiftkey/octokit.net/blob/rewrite-contributing/docs/contributing.md)
+under the `docs` folder.
 
 ### Submitting Changes
 
-You can publish your branch from GitHub for Windows, or run this command from
+You can publish your branch from GitHub Desktop, or run this command from
 the Git Shell:
 
 `git push origin MY-BRANCH-NAME`
 
 Once your changes are ready to be reviewed, publish the branch to GitHub and
-[open a pull request](https://help.github.com/articles/using-pull-requests) 
+[open a pull request](https://help.github.com/articles/using-pull-requests)
 against it.
 
 A few little tips with pull requests:
 
  - prefix the title with `[WIP]` to indicate this is a work-in-progress. It's
    always good to get feedback early, so don't be afraid to open the PR before it's "done".
- - use [checklists](https://github.com/blog/1375-task-lists-in-gfm-issues-pulls-comments) 
+ - use [checklists](https://github.com/blog/1375-task-lists-in-gfm-issues-pulls-comments)
    to indicate the tasks which need to be done, so everyone knows how close you are to done.
  - add comments to the PR about things that are unclear or you would like suggestions on
 
-Don't forget to mention in the pull request description which issue/issues are 
+Don't forget to mention in the pull request description which issue/issues are
 being addressed.
 
 Some things that will increase the chance that your pull request is accepted.
```

Comment:
Some things that will increase the chance that your pull request is accepted~~.~~**:**

(end with colon rather than full stop)